examples/cifar/src/vit.py

```diff
@@ -0,0 +1,139 @@
+# https://github.com/lucidrains/vit-pytorch/blob/main/vit_pytorch/vit_pytorch.py
```

Comment:
following the MIT license, you must include in the file as comment, the full MITlicense 
(see https://github.com/facebookresearch/diffq/blob/main/examples/cifar/src/mobilenet.py#L9)

Reply:
Sure!

Reply:
updated